usaspending_api/recipient/management/sql/recipient_lookup/050_sam_missing_name.sql

```diff
@@ -19,7 +19,7 @@ INSERT INTO public.temporary_restock_recipient_lookup (
 )
 SELECT
   DISTINCT ON (awardee_or_recipient_uniqu, legal_business_name)
-  MD5(UPPER(CONCAT('duns-', awardee_or_recipient_uniqu)))::uuid AS recipient_hash,
+  MD5(COALESCE(UPPER(CONCAT('uei-', uei)), UPPER(CONCAT('duns-', awardee_or_recipient_uniqu))))::uuid AS recipient_hash,
```

Comment:
Use of `CONCAT` with a text and null value will always result in non-null, so you'll get `uei-` as the value and it will never use the `awardee_or_recipient_uniqu` in case of a null `uei`. See if you can figure out how to move the coalesce inward, to work on the individual fields and then do the prefix and MD5 on the result of coalesce.

![image](https://user-images.githubusercontent.com/14953413/129104777-0d252c48-4ef6-4085-80dd-0e3a1c349905.png)